powerline/lib/vcs/git.py

```diff
@@ -102,6 +102,13 @@ class Repository(GitRepository):
 		def ignore_event(path, name):
 			return False
 
+		def stash(self):
+			try:
+				return sum(1 for _ in git.Repository(git_directory(self.directory))
+					.lookup_reference('refs/stash').log())
```

Comment:
`.attr` should not be on the next line, also there are too many calls under `try`. You should use

    try:
        stashref = git.Repository(git_directory(self.directory)).lookup_reference('refs/stash')
    except KeyError:
        return None
    return sum(1 for _ in stashref.log())